rebasehelper/build_helper.py

```diff
@@ -386,6 +390,8 @@ def _build_rpm(cls, srpm, workdir, results_dir):
         output = os.path.join(results_dir, "build.log")
 
         cmd = [cls.CMD, '--rebuild', srpm]
+        if enable_option is not None:
+            cmd.extend(filter(None, enable_option.split(" ")))
```

Comment:
The same code is also in Rpmbuild build tool class. Maybe it would be a good idea to extract the actual code to the BuildToolBase class and call it from there. Or create new common base class for local builders, since these are the only ones for which it makes sense.